troubleshooting.md

```diff
@@ -327,23 +327,23 @@ Organization 'my-org' not found
 
 #### Possible causes
 
-The user creating the CFOrg is not bound to the `korifi-controllers-admin`
-ClusterRole.
+1. The user creating the CFOrg does not have a `RoleBinding` that is bound to the `korifi-controllers-admin` ClusterRole.
+2. The user has a `RoleBinding` but the binding is assigned to a username that is defined in the user's local Kubeconfig but isn't a recognized user on the Kubernetes cluster. 
```

Comment:
```suggestion
2. The user appears to have a `RoleBinding`, but the binding is assigned to a local Kubeconfig user entry that does not match a recognized user on the Kubernetes cluster.
```

Reply:
This one's hard to word. 😅 I suggested a tweak, but not sure myself if that is any clearer. Maybe @HenryBorys can help us out here. 🙂

Reply:
Looks good. I just removed a couple of extra words.